portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1703,6 +1703,91 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	    });
 	    return fetch_promise.promise();
 	}),
+	'getUnsequencedSamplesBasedOnSequencedSampleList': makeCachedPromiseFunction(
+		function(self, fetch_promise) {
+		    var sample_list_ids = self.getCancerStudyIds().map(function(cancer_study_id) {
+			return cancer_study_id + "_sequenced";
```

Comment:
@pieterlukasse @fedde-s I agree with what you're saying.
I'm looking at the `sample_list` table now, and it looks like the sample lists we're interested in can be more robustly picked out as having the category `all_cases_with_mutation_data`. What do you all think of that? @jjgao